### PR TITLE
Add resource parsing to Torque

### PR DESCRIPTION
**Pull request by nhazekam**
Should address #1857. Untested by virtue of not having access to a Torque cluster. @btovar please take a look.

### CODE REVIEW

**Comment by dthain** [comment]
The idea looks about right to me, but the main thing is to run it by the person who is experiencing the problem...

**Comment by nhazekam** [comment]
This only touches a single file and should be able to be cherry picked to wherever we want. I don't know if testing this on master is necessarily the best option. @btovar is there a prefered branch I should base this off?

**Comment by nhazekam** [comment]
Rudimentary testing was done to make sure that changes to the command line were being made, -dall does print out the updated line with resources. 

I agree it is just up to the user for testing at this point.

**Comment by dthain** [comment]
Please reach out and invite him to try it.

**Comment by btovar** [comment]
Looks good to me, assuming that arguments can be repeated and the last overrides the previous. Let's wait to see when it is tested in a torque system.

**Comment by nhazekam** [comment]
@btovar Can you please take a look? I am still going to add a note about what options are used and why, but the functional code is there.

File: batch_job/src/batch_job_cluster.c [hunk 1]
```diff
@@ -99,6 +100,40 @@ static int setup_batch_wrapper(struct batch_queue *q, const char *sysname )
...
+static char *cluster_set_resource_string(struct batch_queue *q, const struct rmsummary *resources)
+{
+	char *cluster_resources = NULL;
+	const char *ignore_mem = hash_table_lookup(q->options, "ignore-mem-spec");
+
+	if(q->type == BATCH_QUEUE_TYPE_TORQUE || q->type == BATCH_QUEUE_TYPE_PBS){
+		char *mem = string_format(",mem=%" PRId64 "mb", resources->memory);
+		char *disk = string_format(",file=%" PRId64 "mb", resources->disk);
+		cluster_resources = string_format(" -l nodes=1:ppn=%" PRId64 "%s%s ", 
+			resources->cores ? resources->cores : 1,
```
Comment (btovar) [inline batch_job/src/batch_job_cluster.c]: Probably we want a > 0 here too, as resources->cores may be -1.

File: doc/makeflow.html [hunk 2]
```diff
@@ -410,14 +410,36 @@ <h2>Table of Contents</h2>
 
 <p>Use the <tt>-T pbs</tt> option to submit jobs to the <a href=http://www.arc.ox.ac.uk/content/pbs>Portable Batch System</a> or compatible systems.</p>
 
+<p>This will add the values for cores, memory, and disk. These values will be added onto <tt>qsub</tt> in this format:<br>
+<tt>-l nodes=1:ppn=$CORES,mem=$MEMORY,file=$DISK</tt></p>
```
Comment (btovar) [inline doc/makeflow.html]: should it be 

mem=${MEMORY}mb  (same for file).

We also want to be explicit in that measurements are in MB.

**Pull request merged**